packages/snjs/lib/services/api/api_service.ts

```diff
@@ -700,6 +712,31 @@ export class SNApiService extends PureService<
     });
   }
 
+  public async getSubscription(
+    userUuid: string
+  ): Promise<HttpResponse | GetSubscriptionResponse> {
+    const url = joinPaths(this.host, Paths.v1.subscription(userUuid));
+    const response = await this.request({
+      verb: HttpVerb.Get,
+      url,
+      authentication: this.session?.authorizationValue,
+      fallbackErrorMessage: messages.API_MESSAGE_FAILED_SUBSCRIPTION_INFO,
+    });
+    this.processResponse(response);
```

Comment:
I think processResponse already happens in `this.request`.

Reply:
Ah good catch, thanks! 🙂